ydb/library/actors/util/ticket_lock.h

```diff
@@ -16,33 +16,34 @@ class TTicketLock : TNonCopyable {
     }
 
     void Release() noexcept {
-        AtomicUi32Increment(&TicketOut);
+        TicketOut.fetch_add(1, std::memory_order_release);
     }
 
     ui32 Acquire() noexcept {
         ui32 revolves = 0;
-        const ui32 ticket = AtomicUi32Increment(&TicketIn) - 1;
-        while (ticket != AtomicLoad(&TicketOut)) {
-            Y_DEBUG_ABORT_UNLESS(ticket >= AtomicLoad(&TicketOut));
+        const ui32 ticket = TicketIn.fetch_add(1, std::memory_order_seq_cst);
```

Comment:
Why is std::memory_order_seq_cst used here instead of std::memory_order_acq_rel?

Reply:
Need to change to `std::memory_order_acq_rel`.

Reply:
It can actually be relaxed, because `TicketIn` is used purely as a unique order counter. Since `Release()` uses release on `TicketOut` only acquire on `TicketOut` can synchronize with `Release()`.